src/Identity/v3/Models/Token.php

```diff
@@ -43,6 +44,8 @@ class Token extends OperatorResource implements Creatable, Retrievable, \OpenSta
     protected $resourceKey = 'token';
     protected $resourcesKey = 'tokens';
 
+    protected $cacheCredential;
```

Comment:
wondering what the best name for this is. `cachedToken`, `cachedResponse`, `cachedArray`?

"credential" seems quite ambiguous with ref to tokens

Reply:
we should also add a description saying that the array's structure is a modified version of the POST response, since it has an "id" key added